### PR TITLE
[JOLINT-100 ] loader skeleton

### PR DESCRIPTION
**Pull request by idamakela**
New npm package installed

### CODE REVIEW

File: app/dashboard/_components/cards/WeeklyInclusionScore/WeeklyInclusionScore.tsx [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use client'
 
-import { useCallback, useEffect, useState } from 'react'
+import { Suspense, useCallback, useEffect, useState } from 'react'
```
Comment (idamakela) [inline app/dashboard/_components/cards/WeeklyInclusionScore/WeeklyInclusionScore.tsx:3]: remove suspense

File: app/dashboard/_components/cards/InclusionOfNewHires/InclusionOfNewHires.tsx [hunk 2]
```diff
@@ -1,16 +1,21 @@
...
 import { getUnique } from '@/utils/getUnique'
 import { getCurrentMetrics } from '@/utils/getCurrentMetrics'
 import Card from '@/components/Card'
 import CardHeader from '@/app/dashboard/_components/CardHeader'
 import InclusionOfNewHiresChart from '@/app/dashboard/_components/cards/InclusionOfNewHires/InclusionOfNewHiresChart'
+import Loading from '@/app/dashboard/_components/Loading'
 import type { metrics } from '@/utils/types'
 
 const InclusionOfNewHires = () => {
+  // works on rezise but not on initialization
```
Comment (idamakela) [inline app/dashboard/_components/cards/InclusionOfNewHires/InclusionOfNewHires.tsx:16]: remove comment

**Comment by ell-ska** [comment]
I finished this ticket since Ida had some other things to deal with

**Review by xxrobone — APPROVED** [review]
Skeleton loader, code looking good, i see no errors

**Review by xxrobone — APPROVED** [review]
Skeleton loader, code looking good, i see no errors

**Comment by ell-ska** [comment]
ready for testing!

**Comment by xxrobone** [comment]
Testing works good.

**Comment by DavidDyberg** [comment]
### Dev test
Looks good, ready for production!

**Pull request merged**